src/repr/src/relation_and_scalar.proto

```diff
@@ -110,5 +114,6 @@ message ProtoScalarType {
         google.protobuf.Empty UInt32 = 30;
         google.protobuf.Empty UInt64 = 31;
         google.protobuf.Empty MzTimestamp = 32;
+        ProtoRange Range = 33;
```

Comment:
@aalexandrov has requested that he is notified about any protobuf changes . So I am adding him as a reviewer, even though nothing funky seems to be going on here , that is, there are no deletions in the protobuf definitions.